bayes/inference_problem.py

```diff
@@ -65,7 +92,7 @@ def loglike(self, number_vector):
         return log_like
 
 
-class VariationalBayesProblem(InferenceProblem):
+class VariationalBayesProblem(InferenceProblem, VariationalBayesModelError):
```

Comment:
The `VariationalBayesModelError` is the interface for the VB algorithm. That makes it clear that `VariationalBayesProblem` can directly be plugged into our VB algorithm. As typical for the duck-typing of python, we could remove this superclass without any problems. I just thought, you liked explicit interfaces. 
You decide to keep or remove it. Or should I rename it to `VariationalBayesInterface` or something?

Reply:
From my point of view, the model error is a part of the inference problem (similar to the non-VB models) with e.g. the noise groups being other members of the inference problem. Putting this together seems to contradict this concept.

Reply:
There is a _huge_ difference between the `ModelErrorInterface`:
* no arguments (because we depend on `self.parameter_list`)
* returns `dict {sensor_key: numpy_vector}`

and the `VariationalBayesModelError`:
* list of numbers or numpy_vector as arguments
* returns `dict {noise_key: numpy_vector}`

Even though the return type is technically the same, the inputs/arguments are different. I'll rename the latter one to `VariationalBayesInterface` to remove any verbal connection to the model error.